### PR TITLE
Fix issues with lint .js files in typescript environments

### PR DESCRIPTION
**Pull request by dmfilipenko**
### 🔦 Summary
1. Added a list of js files to include a rule to make it visible for a linter. 

https://github.com/wix/yoshi/issues/1091

### CODE REVIEW

**Comment by yurynix** [comment]
Thank you for this PR!
Regarding adding:
```
"./src/**/*.js"
```
That sounds too broad to me, should we allow making arbitrary files in a TS project?
I think @ranyitz meant in his issue only include the configuration files that must remain in JS.
what do you think @dmfilipenko ?

**Comment by ranyitz** [comment]
Thanks @yurynix, that's correct. Furthermore, it's important to say that most of those files are not in the `src` directory, and therefore won't be linted. (`index.js`, `jest-yoshi.config.js` etc ...)

File: packages/create-yoshi-app/templates/fullstack/typescript/.vscode/settings.json [hunk 1]
```diff
@@ -35,5 +35,6 @@
   "html.format.extraLiners": "",
   "files.insertFinalNewline": true,
   "eslint.autoFixOnSave": true,
```
Comment (ranyitz) [inline packages/create-yoshi-app/templates/fullstack/typescript/.vscode/settings.json]: I know that it's not related to the PR, but it hurts looking at it.
```suggestion

```

**Comment by dmfilipenko** [comment]
@ranyitz, please review

**Comment by ranyitz** [comment]
@dmfilipenko That's the error from the CI. It's not really emitting files there, but it seems that TypeScript is not allowing it, so it makes sense to just add the `outDir` configuration anyway.

image

**Pull request merged**